diff --git a/node_modules/cross-zip/index.js b/node_modules/cross-zip/index.js
index 94b6142..24cff8e 100644
--- a/node_modules/cross-zip/index.js
+++ b/node_modules/cross-zip/index.js
@@ -10,6 +10,8 @@ const cp = require('child_process')
 const fs = require('fs')
 const os = require('os')
 const path = require('path')
+const removeDir = fs.rm ? fs.rm : fs.rmdir
+const removeDirSync = fs.rmSync ? fs.rmSync : fs.rmdirSync
 
 function zip (inPath, outPath, cb) {
   if (!cb) cb = function () {}
@@ -46,7 +48,7 @@ function zip (inPath, outPath, cb) {
   // Windows zip command does not overwrite existing files. So do it manually first.
   function doZip () {
     if (process.platform === 'win32') {
-      fs.rmdir(outPath, { recursive: true, maxRetries: 3 }, doZip2)
+      removeDir(outPath, { recursive: true, maxRetries: 3 }, doZip2)
     } else {
       doZip2()
     }
@@ -72,7 +74,7 @@ function zipSync (inPath, outPath) {
       fs.writeFileSync(path.join(tmpPath, path.basename(inPath)), inFile)
       inPath = tmpPath
     }
-    fs.rmdirSync(outPath, { recursive: true, maxRetries: 3 })
+    removeDirSync(outPath, { recursive: true, maxRetries: 3 })
   }
   const opts = {
     cwd: path.dirname(inPath),
